pkg/cmd/builder/build.go

```diff
@@ -419,6 +419,29 @@ func getDigestFromMetaFile(path string) (string, error) {
 	return digest, nil
 }
 
+func isMatchingRuntimePlatform(platform string) bool {
+	p, err := platforms.Parse(platform)
+	if err != nil {
+		return false
```

Comment:
Should we log this error?
```suggestion
                log.L.WithError(err).Errorf("failed to parse platform")
		return false
```

Reply:
i don't think its important to log here, as the Parse will be called elsewhere and be handled. Here it just important to figure out if the image is shareable, if any thing goes wrong it will attempt to create a non-shareable image.